setup.cfg

```
@@ -29,8 +29,7 @@ include_package_data = True
 install_requires =
     napari>=0.4.13
     scikit-image
-    tqdm
-    pymmcore-plus >=0.1.8
+    pymmcore-plus >=0.2.0
```

```suggestion
    pymmcore-plus >=0.2.1
```

(releasing now, with the fix to setPosition)

---

done